scbd/absch.cbd.int
# chm-418 virtual libaray resource  edit page

### General description
migrate virtual library resource report edit page
change some logic and add gbf target part

### Designs
same as absch project

### Testing instructions
submit-> virtual library resource 
use finished review page help to check.

<img width="851" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/0b5319db-3ccb-46bd-9fd0-a01be083cb56">
